src/config/constants.ts

```diff
@@ -808,3 +808,14 @@ export enum CONFIGURATION_TYPES {
     NAMESPACE = 'NAMESPACE',
     DESCRIPTION = 'DESCRIPTION',
 }
+
+export enum DEPLOYMENT_APP_TYPE {
```

Comment:
DEPLOYMENT_APP_TYPE should be DeploymentAppType

Reply:
Done